src/server.js

```diff
@@ -28,6 +28,10 @@ app.use(cors({
   exposedHeaders: ['Content-Range', 'X-Content-Range'],
 }));
 
+//allowing cors on all routes and all request in  pre-flight phase
+
+app.options('*' , cors());
```

Comment:
There's already a `cors` object being used on line `27`. You should use that object to configure the CORS settings so we avoid having duplicate objects.